subprojects/robotpy-wpilib/wpilib/iterativerobotpy.py

```diff
@@ -0,0 +1,237 @@
+import hal
+import ntcore
+import wpilib
+import wpilib._impl
+import wpilib._impl.report_error
+
+
+from enum import IntEnum
+
+
+class IterativeRobotMode(IntEnum):
```

Comment:
nit: this is never used as an integer value, so we can use `enum.Enum` directly instead.

```suggestion
import enum


class IterativeRobotMode(enum.Enum):
```